source/manuals/programming-languages/nodejs/testing.html.md.erb

```diff
@@ -0,0 +1,80 @@
+---
+title: Testing Node.js Apps at GDS
+---
+
+# <%= current_page.data.title %>
+
+## Libraries
+
+At GDS we use Mocha as our test runner and Chai as our assertion library.
+These form part of a larger group of commonly used testing libraries, a non-exhaustive list of libraries we use can be found below.
```

Comment:
More of a general question, does there need to be any justification over why we use these libraries over others that are available? Should a section be added around libraries that we should definitely avoid since we have had issues in the past?